source/renderer/app/components/wallet/tokens/wallet-token/WalletToken.js

```diff
@@ -0,0 +1,89 @@
+// @flow
```

Comment:
This file was result from refactoring [WalletToken.js](https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/components/wallet/tokens/WalletToken.js)